source/iface/command-accessor.rst

```diff
@@ -6,281 +6,274 @@
+           handler& commandGroupHandlerRef, const property_list& propList = {})
 
+  template <typename AllocatorT, typename TagT>
+  accessor(buffer<DataT, Dimensions, AllocatorT>& bufferRef,
+           handler& commandGroupHandlerRef, TagT tag,
+           const property_list& propList = {})
 
-Construct an accessor for a buffer.
+  template <typename AllocatorT>
+  accessor(buffer<DataT, Dimensions, AllocatorT>& bufferRef,
+           range<Dimensions> accessRange, const property_list& propList = {})
 
-Programs typically find it more convenient to use
-:ref:`buffer-get_access` to create an accessor for a buffer.
+  template <typename AllocatorT, typename TagT>
+  accessor(buffer<DataT, Dimensions, AllocatorT>& bufferRef,
+           range<Dimensions> accessRange, TagT tag,
+           const property_list& propList = {})
 
-.. rubric:: Template parameters
+  template <typename AllocatorT>
+  accessor(buffer<DataT, Dimensions, AllocatorT>& bufferRef,
+           range<Dimensions> accessRange, id<Dimensions> accessOffset,
+           const property_list& propList = {})
 
-==============  ===
-``AllocatorT``  Type of allocator for buffer element
-==============  ===
+  template <typename AllocatorT, typename TagT>
+  accessor(buffer<DataT, Dimensions, AllocatorT>& bufferRef,
+           range<Dimensions> accessRange, id<Dimensions> accessOffset, TagT tag,
+           const property_list& propList = {})
 
-.. rubric:: Parameters
+  template <typename AllocatorT>
+  accessor(buffer<DataT, Dimensions, AllocatorT>& bufferRef,
+           handler& commandGroupHandlerRef, range<Dimensions> accessRange,
+           const property_list& propList = {})
 
-==========================  ===
-``bufferRef``               Associate accessor with this buffer
-``commandGroupHandlerRef``  Associate accessor with this handler
-``propList``                `sycl::accessor (buffer) properties`_
-``accessRange``             Dimensions of data to be accessed
-``accessOffset``            Coordinates of origin of data
-==========================  ===
+  template <typename AllocatorT, typename TagT>
+  accessor(buffer<DataT, Dimensions, AllocatorT>& bufferRef,
+           handler& commandGroupHandlerRef, range<Dimensions> accessRange,
+           TagT tag, const property_list& propList = {})
 
-``is_placeholder``
-==================
+  template <typename AllocatorT>
+  accessor(buffer<DataT, Dimensions, AllocatorT>& bufferRef,
+           handler& commandGroupHandlerRef, range<Dimensions> accessRange,
+           id<Dimensions> accessOffset, const property_list& propList = {})
 
-::
+  template <typename AllocatorT, typename TagT>
+  accessor(buffer<DataT, Dimensions, AllocatorT>& bufferRef,
+           handler& commandGroupHandlerRef, range<Dimensions> accessRange,
+           id<Dimensions> accessOffset, TagT tag,
+           const property_list& propList = {})
 
-  constexpr bool is_placeholder() const;
+Constructors of the accessor class.
```

Comment:
Maybe it's nice to have a detailed description of constructors in the reference guide. You can take structure from buffer constructors section